v21.1/orchestrate-cockroachdb-with-kubernetes-insecure.md

```diff
@@ -103,12 +103,16 @@ To start your CockroachDB cluster, you can either use our StatefulSet configurat
 
 To shut down the CockroachDB cluster:
 
-1. Delete all of the resources you created, including the logs and remote persistent volumes:
+1. Delete all of the resources you created, including the logs:
+
+    {{site.data.alerts.callout_danger}}
+    This does not include deleting the persistent volumes that were attached to the pods. If you want to delete the persistent volumes and free up the storage used by CockroachDB, be sure you have a backup copy of your data. Data **cannot** be recovered once the persistent volumes are deleted. For more information, see the [Kubernetes documentation](https://kubernetes.io/docs/tasks/run-application/delete-stateful-set/#persistent-volumes).
```

Comment:
I am not sure why "all of the resources you created" would not include data on persistent volumes attached to the pods. This might make sense to a person who's gotten to this point, but I'm new to CRDB in a Kubernetes environment, so the meaning is not obvious to me. So LGTM, if you think this meets the needs of the intended audience after considering this feedback and maybe you can clarify this distinction for me, regardless.

Reply:
Thanks for this! It's potentially confusing for everyone IMO. I think I'll amend it by deleting the word "all".